pkg/api/grids/grids.go

```diff
@@ -85,9 +86,17 @@ func ReturnSingleGrid(w http.ResponseWriter, r *http.Request) {
 
 	glog.V(4).Info("GET /grids/" + key)
 
+	foundApplication := false
 	for _, grids := range Grids {
 		if grids.ID == key {
+			foundApplication = true
```

Comment:
Should the logic break out of the loop once found is true?

Reply:
Hey Mike, added break statement, Tnx